src/hugchat/prompt.py

```diff
@@ -0,0 +1,674 @@
+# Pre-defined Prompts for the hugchat command line interface.
+
+def get_roles():
+    """Return a list containing just the names of all defined roles."""
+    return [role['role'] for role in PROMPTS.values()]
+
+def handle_prompt():
+    # Print a menu allowing users to select a role
+    print("\nAvailable Roles:\n")
+    for index, role in enumerate(get_roles(), start=1):
+        print(f"[{index}.{role}]", end=", ")
+
+    # Request the user's selection
+    choice = None
+    while not isinstance(choice, int) or choice < 1 or choice > len(PROMPTS):
+        try:
+            choice = int(input("\nPlease make a valid selection: ")) - 1
+        except ValueError:
+            pass
+
+    # Obtain the selected role and associated details
+    chosen_role = get_roles()[choice]
+    prompt = next((p for p in PROMPTS.values() if p['role'].lower() == chosen_role.lower()), None)
+
+    if prompt:
+        role = prompt['role']
+        description = prompt['description']
+        return role, description
+
+    raise Exception(f"No such role found ({chosen_role})")
+
+# Prompt list (in dict)
+PROMPTS ={
+    1:{
+        'id': 1,
+        'role':"scientfic",
+        'description':("desc here"),
+      },
+    2:{
+        'id': 2,
+        'role':"SEO Prompt",
+        'description':("Using WebPilot, create an outline for an article that will be 2,000 words on the keyword \'Best SEO prompts\' based on the top 10 results from Google. Include every relevant heading possible. Keep the keyword density of the headings high. For each section of the outline, include the word count. Include FAQs section in the outline too, based on people also ask section from Google for the keyword. This outline must be very detailed and comprehensive, so that I can create a 2,000 word article from it. Generate a long list of LSI and NLP keywords related to my keyword. Also include any other words related to the keyword. Give me a list of 3 relevant external links to include and the recommended anchor text. Make sure they’re not competing articles. Split the outline into part 1 and part 2."),
```

Comment:
Why do we need a tuple as the value for this? It seems like you are only using the first option in your code anyway. A key such as "prompt" might also be clearer.